e2e-tests/gef/cypress/integration/provided-facility.spec.js

```diff
@@ -96,8 +92,8 @@ context('Provided Facility Page', () => {
       providedFacility.mainHeading().contains('contingent');
       providedFacility.mainHeading().should('not.contain', 'cash');
       providedFacility.form();
-      providedFacility.hiddenFacilityType().should('not.be', 'visible');
-      providedFacility.detailsOther().should('not.be', 'visible');
+      providedFacility.hiddenFacilityType().should('not.be.visible');
+      providedFacility.detailsOther();
```

Comment:
do we need to add a .should('not.be.visible'); check here?

Reply:
Problem is with this - it is visible. The original check is wrong so this fails. I think from what I could see looking at the component there is no way this would be hidden in the circumstances I believe. Sorry my memory of each case is not as clear